script/contracts/L2/L2Airdrop.s.sol

```diff
@@ -27,11 +30,6 @@ contract L2AirdropScript is Script {
         assert(newOwnerAddress != address(0));
         console2.log("L2 Airdrop owner address: %s (after ownership will be accepted)", newOwnerAddress);
 
-        // address to which the airdrop wallet is set
-        address airdropWalletAddress = vm.envAddress("L2_AIRDROP_WALLET_ADDRESS");
-        assert(airdropWalletAddress != address(0));
-        console2.log("L2 Airdrop wallet address: %s", airdropWalletAddress);
-
         // get L2LiskToken contract address
         Utils.L2AddressesConfig memory l2AddressesConfig = utils.readL2AddressesFile();
```

Comment:
I see that the `l2addresses.json` file is in the .gitignore so it is only used locally. Should we also upload it to github so we could check the correctness of all the used addresses? I want to make sure that the addresses agree with the ones [here](https://github.com/LiskHQ/lisk-contracts/blob/3195511f3ecb1b21ca2db125f3fb1d6d8f58c3e1/deployment/addresses/mainnet/addresses.md).
Or is there some privacy reason why `l2addresses.json` are not uploaded?

Reply:
This will be actually resolved in another PR, which I've just created: https://github.com/LiskHQ/lisk-contracts/pull/149

Reply:
I've also assigned you @sergeyshemyakov as a reviewer.